src/services/nft/index.ts

```diff
@@ -96,10 +96,15 @@ export const getNftOwnedByAddress = async (
   collectionAddress: string,
   walletAddress: string,
 ) => {
-  if (chainId === Chains.mainnet || chainId === Chains.arbitrum) {
+  if (chainId === Chains.mainnet || chainId === Chains.arbitrum || chainId === Chains.polygon) {
     const settings = {
       apiKey: getTokenProvider(chainId),
-      network: chainId === Chains.mainnet ? Network.ETH_MAINNET : Network.ARB_MAINNET,
+      network:
+        chainId === Chains.mainnet
```

Comment:
It's probably better to create a `getAlchemyNetworkId()` util function. Would be good to move away from the ternary operator.